projects/ion/src/lib/notification/notification.component.spec.ts

```diff
@@ -0,0 +1,135 @@
+import { fireEvent, render, screen } from '@testing-library/angular';
+import { StatusType } from '../core/types';
+import { IonIconComponent } from '../icon/icon.component';
+import {
+  NotificationComponent,
+  NotificationProps,
+} from './notification.component';
+
+const defaultNotification = {
+  title: 'Editado',
+  message: 'cadastro',
+  type: 'success' as StatusType,
```

Comment:
Since "type" has a default value, you shouldn't have to test the case when the user does not pass the type parameter?

Reply:
little foot hahaha